### DISCUSSION TITLE
v7 development

### DISCUSSION THREAD

**Discussion by mnigh** [nuwave/lighthouse#2556] [Q&A] [answered]
There is a [label](https://github.com/nuwave/lighthouse/pulls?q=is%3Apr+is%3Aopen+label%3A7.x) and an [upgrade guide](https://github.com/nuwave/lighthouse/blob/master/UPGRADE.md#v6-to-v7) for v7 but no releases, branches, or tags that I was able to find.

Is there a timeline for its release?

**Comment by spawnia** [comment, marked as answer]
No.
